src/field_map_change.c

```diff
@@ -79,15 +80,15 @@
 FS_EXTERN_OVERLAY(overlay23);
 
 typedef struct {
-    int unk_00;
-    Location unk_04;
+    int state;
+    Location nextLocation;
     UnkStruct_ov5_021D432C *unk_18;
-} UnkStruct_02053900;
+} MapChangeData;
 
 typedef struct {
-    int unk_00;
-    Location unk_04;
-} UnkStruct_02053A80;
+    int state;
+    Location nextLocation;
+} MapChangeSubData;
```

Comment:
**polish:** Canonize these struct names (`struct X { ...`). Applies to other instances in the file.